<?xml version="1.0"?>
<launch>    
	<node name="object_server" pkg="usydrx_navigation" type="object_server2.py" output="screen"/>

    <!-- <node pkg="usyd_vrx_vision" type="object_classifier.py" name="object_classifier" output="screen"/> -->

    <!-- <node name="object_visualiser" pkg="usydrx_navigation" type="object_to_visualisation.py" output="screen">
        <remap from="/wamv/classified_objects" to="/wamv/unclassified_objects"/>
    </node> -->
</launch>

 